From b8b3df6dea709fcc32644547aa24fcaca01512be Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 5 Apr 2018 17:35:21 +0300
Subject: [PATCH 127/706] enetc: tsn: Fix module linkage issue when TSN enabled

Fixes following link error:
ERROR: "enetc_tsn_init"
[drivers/net/ethernet/freescale/enetc/fsl-enetc.ko] undefined!
scripts/Makefile.modpost:91: recipe for target '__modpost' failed
make[1]: *** [__modpost] Error 1
Makefile:1215: recipe for target 'modules' failed
make: *** [modules] Error 2

Fixes: 32c2127 ("enetc-tsn: add enetc tsn function support")

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 88ad0e7ea3bb3c23bdb74b9f99ce812570f1c85a)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/Makefile b/drivers/net/ethernet/freescale/enetc/Makefile
index 7450801a692d..dbecaf2b3793 100644
--- a/drivers/net/ethernet/freescale/enetc/Makefile
+++ b/drivers/net/ethernet/freescale/enetc/Makefile
@@ -1,6 +1,6 @@
 obj-$(CONFIG_FSL_ENETC) += fsl-enetc.o
 fsl-enetc-$(CONFIG_FSL_ENETC) += enetc.o enetc_cbdr.o enetc_ethtool.o
-obj-$(CONFIG_ENETC_TSN) += enetc_tsn.o
+fsl-enetc-$(CONFIG_ENETC_TSN) += enetc_tsn.o
 fsl-enetc-$(CONFIG_PCI_IOV) += enetc_msg.o
 fsl-enetc-objs := enetc_pf.o $(fsl-enetc-y)
 
-- 
2.17.1

